### DISCUSSION TITLE
cart issue

### DISCUSSION THREAD

**Discussion by sai-pranay-teja** [learndevopsonline/learndevopsonline#833] [Q&A]
Hi @r-devops 

I'm unable to add the products into the cart
![image](https://github.com/learndevopsonline/learndevopsonline/assets/36685961/a7cc7b0d-51bf-4655-9c2d-17824bb3f6e5)

The code I'm using old roboshop shell code WET, with all normal commands

The recordname for frontend: http://frontend.practise-devops.online/
And cart IP address: 18.233.167.154

Please let me know what's the issue

Thank you

**Comment by Arafique458** [comment]
Are you getting any error message when you run the code if not i would recommend restarting cart server and also nginx service along with frontend server. This happened to me as well.

**Reply by sai-pranay-teja** [reply]
@Arafique458 Well, I didn't exactly got an error, but I got like this when I re-run the cart component code
![image](https://github.com/learndevopsonline/learndevopsonline/assets/36685961/91839a11-e380-4761-a4a6-05da1c97c222)

And I was able to check the status and Its running, and I restarted again, for both cart and nginx service. No luck, it wasnt resolved
